resources/lib/play.py

```diff
@@ -41,8 +40,11 @@ def play(url):
         # set in program.url, otherwise we fetch it separately
         if p.get_url():
             stream_url = p.get_url()
+            sub_url = None
         else:
-            stream_url = comm.get_stream(p.id)
+            stream_info = comm.get_stream(p.id)
+            stream_url = stream_info['url']
+            sub_url = stream_info['subtitles']
```

Comment:
set `sub_url = None` above the if, then do `if 'subtitles' in stream_info: sub_url = stream_info['subtitles']`